### ISSUE TITLE
Exception while training, won't generate CKPT

### ISSUE THREAD

**Issue by YakuzaSuske** [d8ahazard/sd_dreambooth_extension#340]
![image](https://user-images.githubusercontent.com/63294141/204132061-5cc8e6f1-244d-41a8-aa13-d4315610a7b3.png)
```
venv "C:\SUPER_SD_2.0\stable-diffusion-webui-master\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: b5050ad2071644f7b4c99660dc66a8a95136102f
Installing requirements for Web UI
Checking Dreambooth requirements...
Checking/upgrading existing torch/torchvision installation
Checking torch and torchvision versions
Dreambooth revision is a66a2d6bca536207ca3154324f73a5cd43a65cb4
```

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes

**Have you updated Dreambooth to the latest revision?**
Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?**

OK

**Describe the bug**

After training, the program won't generate a ckpt file saying it's missing a VAE path and Half checkpoint, it has never happened to me before this new update so i'm a bit confused.

```
***** Running training *****
  Num examples = 21
  Num batches each epoch = 21
  Num Epochs = 400
  Instantaneous batch size per device = 1
  Total train batch size (w. parallel, distributed & accumulation) = 1
  Gradient Accumulation steps = 1
  Total optimization steps = 8400
   Training settings: CPU: False Adam: True, Prec: fp16, Grad: True, TextTr: True EM: False, LR: 1e-06
 Allocated: 3.8GB
 Reserved: 3.9GB

Steps:   0%|                                   | 5/8400 [00:07<2:12:12,  1.06it/s, loss=0.00751, lr=1e-6, vram=5.7/9.5] Step 5 completed.
 Allocated: 5.7GB
 Reserved: 9.5GB

Steps: 100%|██████████████████████████████████▉| 8398/8400 [1:13:14<00:01,  1.89it/s, loss=0.11, lr=1e-6, vram=5.7/9.5]Popping concept:
 Exception while training: ''
 Allocated: 5.7GB
 Reserved: 9.5GB

Traceback (most recent call last):
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\extensions\sd_dreambooth_extension\dreambooth\train_dreambooth.py", line 965, in main
    for step, batch in enumerate(train_dataloader):
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\venv\lib\site-packages\accelerate\data_loader.py", line 384, in __iter__
    next_batch = next(dataloader_iter)
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\venv\lib\site-packages\torch\utils\data\dataloader.py", line 681, in __next__
    data = self._next_data()
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\venv\lib\site-packages\torch\utils\data\dataloader.py", line 721, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\venv\lib\site-packages\torch\utils\data\_utils\fetch.py", line 49, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\venv\lib\site-packages\torch\utils\data\_utils\fetch.py", line 49, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\extensions\sd_dreambooth_extension\dreambooth\SuperDataset.py", line 192, in __getitem__
    self.class_dict.pop(concept_key)
KeyError: ''
 CLEANUP:
 Allocated: 5.7GB
 Reserved: 9.5GB

 Cleanup completed.
 Allocated: 5.7GB
 Reserved: 9.3GB

 Cleanup Complete.
 Allocated: 5.7GB
 Reserved: 9.3GB

Steps: 100%|██████████████████████████████████▉| 8398/8400 [1:13:15<00:01,  1.91it/s, loss=0.11, lr=1e-6, vram=5.7/9.5]
 Training completed, reloading SD Model.
 Allocated: 0.0GB
 Reserved: 3.8GB

Memory output: {'Training completed, reloading SD Model.': '0.0/3.8GB'}
 Restored system models.
 Allocated: 2.2GB
 Reserved: 3.9GB

Returning result: Training finished. Total lifetime steps: 8398

Error completing request
Arguments: ('NeytiriV3',) {}
Traceback (most recent call last):
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\modules\ui.py", line 185, in f
    res = list(func(*args, **kwargs))
  File "C:\SUPER_SD_2.0\stable-diffusion-webui-master\webui.py", line 56, in f
    res = func(*args, **kwargs)
TypeError: compile_checkpoint() missing 2 required positional arguments: 'vae_path' and 'half_checkpoint'
```

**Environment**
Windows, RTX 3080Ti

**Screenshots/Config**
![image](https://user-images.githubusercontent.com/63294141/204131984-46863495-8e08-4d99-b30c-d913b9114d17.png)

**Comment by YakuzaSuske** [comment]
New model has the same problem
![image](https://user-images.githubusercontent.com/63294141/204139238-92158b67-1ab2-411b-8437-da2b7552d0b0.png)

**Comment by cerspense** [comment]
Same issue here. Newest WebUI, newest dreambooth plugin, SD 1.5

**Comment by axonaxis** [comment]
The same bug as everyone else after dreambooth extension update yesterday, ![Dreambooth](https://user-images.githubusercontent.com/30280846/204232913-7970ef23-4e7d-481a-b088-d7ef45fb32c5.png)

**Comment by YakuzaSuske** [comment]
To fix the bug i had to revert to [c589a35](https://github.com/d8ahazard/sd_dreambooth_extension/commit/c589a3596ade64228de8a7851f50c2470c7a76aa)

**Comment by LiamDavisonGates** [comment]
How do you revert the extension to a specific commit?

**Comment by axonaxis** [comment]
thank you for sharing.

**Comment by d8ahazard** [comment]
Resolved with 1913bc7.
